templates/todo/projects/nodejs-mongo-aks/README.md

```diff
@@ -0,0 +1,210 @@
+# ToDo Application with a Node.js API and Azure Cosmos DB API for MongoDB on Azure Kubernetes Service (AKS)
+
+[![Open in Remote - Containers](https://img.shields.io/static/v1?label=Remote%20-%20Containers&message=Open&color=blue&logo=visualstudiocode)](https://vscode.dev/redirect?url=vscode://ms-vscode-remote.remote-containers/cloneInVolume?url=https://github.com/azure-samples/todo-nodejs-mongo)
+
+A complete ToDo application that includes everything you need to build, deploy, and monitor an Azure solution. This application uses the Azure Developer CLI (azd) to get you up and running on Azure quickly, React.js for the Web application, Node.js for the API, Azure Cosmos DB API for MongoDB for storage, and Azure Monitor for monitoring and logging. It includes application code, tools, and pipelines that serve as a foundation from which you can build upon and customize when creating your own solutions.
+
+Let's jump in and get the ToDo app up and running in Azure. When you are finished, you will have a fully functional web app deployed on Azure. In later steps, you'll see how to setup a pipeline and monitor the application.
+
+<img src="assets/web.png" width="75%" alt="Screenshot of deployed ToDo app">
+
+<sup>Screenshot of the deployed ToDo app</sup>
+
+### Prerequisites
+
+The following prerequisites are required to use this application. Please ensure that you have them all installed locally.
+
+- [Azure Developer CLI](https://aka.ms/azure-dev/install)
+  - Windows:
+    ```powershell
+    powershell -ex AllSigned -c "Invoke-RestMethod 'https://aka.ms/install-azd.ps1' | Invoke-Expression"
+    ```
+  - Linux/MacOS:
+    ```
+    curl -fsSL https://aka.ms/install-azd.sh | bash
+    ```
+- [Azure CLI (2.38.0+)](https://docs.microsoft.com/cli/azure/install-azure-cli)
+- [Node.js with npm (16.13.1+)](https://nodejs.org/) - for API backend and Web frontend
+- [Git (2.36.1+)](https://git-scm.com/)
+$PLACEHOLDERIACTOOLS
```

Comment:
I made a change to undo the IaC substitution. I've noticed that changing an IaC doesn't just change tooling, it changes many different things, and I'd like to avoid us if possible. It's easier to review and maintain readme files that are specific to each actual template repository.

Reply:
This template doesn't even use the substitutions (i can remove that $var), but I actually like the concept of having small replaceable components vs maintaining N number of readme's.

Reply:
There's a real problem in not realizing what needs to be substituted differently across readmes. For example, you encountered and fixed the issue that we didn't substitute the actual command correctly in `terraform` template. Stuff like that are easily missed, and it's hard for me to look the other way in lieu of the actual evidence..

We already N readmes coming from each template repo, it seems like N + m isn't a bad tradeoff. That's my current perspective on it.